src/main/java/utility/Choice.java

```diff
@@ -0,0 +1,5 @@
+package utility;
+
+public enum Choice {
```

Comment:
I like how you used an enum class for the different commands

Reply:
Agreed! It's clean!